test/spec/functions/compact.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var compact = require('../../../src/functions/compact');
+
+test('compact removes falsy values from arrays', function() {
+  expect(compact([2])).toEqual([2]);
+  expect(compact([2, false, null, undefined])).toEqual([2]);
+  expect(compact([2, '45', 44])).toEqual([2, '45', 44]);
```

Comment:
`0` and `''` are missing from the test (see [Lodash test usage](https://lodash.com/docs/4.17.11#compact)).